fetch.bs

```diff
@@ -4103,6 +4103,23 @@ steps:
    <var>processBody</var>, <var>processBodyError</var>, and <var>fetchParams</var>'s
    <a for="fetch params">task destination</a>.
   </ol>
+
+ <li><p>If <var>fetchParams</var>'s <a for="fetch params">process response done</a> is non-null, and
+ <var>response</var> is a <a>network error</a>, then:
+
+ <ol>
+  <li><p>Set <var>response</var>'s <a for=response>URL list</a> to a <a for=/>list</a> containing
+  <var>fetchParams</var>'s <a for="fetch params">request</a>'s <a for=request>URL</a>.
+
+  <li><p>Set <var>response</var>'s <a for=response>timing info</a> to the result of
+  <a>creating an opaque timing info</a> for <var>fetchParams</var>'s
+  <a for="fetch params">timing info</a>.
```

Comment:
This would mean that only a network error through this callback is initialized this way. I think we want to do this at the top of "fetch finale".

Reply:
Right, fixed.